helm-charts/charts/kube-starrocks/charts/operator/values.yaml

```diff
@@ -33,6 +33,8 @@ starrocksOperator:
     # image sliced by "repository:tag"
     repository: starrocks/operator
     tag: v1.8.6
+  args:
```

Comment:
not necessary?

Reply:
Operator has some args, like -v for debug log, --namespace

Reply:
I think the namespace has already exposed?. debug log is just for troubleshoot? the example --elect-leader is confusing.

Reply:
We can hide the --elect-leader in template.

Reply:
I have remove the args from values.yaml